cmd/backup-manager/app/backup/manager.go

```diff
@@ -328,19 +328,42 @@ func (bm *Manager) performBackup(ctx context.Context, backup *v1alpha1.Backup, d
 			}
 			errs = append(errs, err)
 			klog.Errorf("cluster %s reset tikv GC life time to %s failed, err: %s", bm, oldTikvGCTime, err)
+
+			// Calculate the backup size for failed backup job
+			backupSize, err := util.CalcVolSnapBackupSize(ctx, backup.Spec.StorageProvider)
```

Comment:
Is this work for non-snapshot based backup?

Reply:
Good catch! I missed non ebs backup